ext/fileinfo/fileinfo.c

```diff
@@ -622,6 +630,36 @@ PHP_FUNCTION(mime_content_type)
 }
 /* }}} */
 
+/* {{{ proto string finfo_magic_file_get()
+   Return path for the magic file */
+PHP_FUNCTION(finfo_magic_file_get)
+{
```

Comment:
I'd suggest to keep the word order same as in method name, finfo_get_magic_file. Easier to memorize.